From ae3896a28f6a90589f2103a53baae4ac61daf7a8 Mon Sep 17 00:00:00 2001
From: Bai Ping <ping.bai@nxp.com>
Date: Fri, 14 Apr 2017 15:57:41 +0800
Subject: [PATCH 2922/5242] MLK-14697 ARM: dts: imx: update the setpoint data
 of imx6sll

commit  7d6adcba24a404f7c1d0d00a590cc569a0c5a1d5 from
https://source.codeaurora.org/external/imx/linux-imx.git

According to the latest datasheet(Rev.0 4/2017), The voltage of
996MHz should be updated to 1.23V. For NXP's Pfuze PMIC chip, the
minimum voltage step is 25mV, we need to set the voltage of 996MHz
to 1.25V. In order to cover board tolerance and IR drop, we add
25mV margin. Then the 996MHz setpoint voltage is 1.275V.

Signed-off-by: Bai Ping <ping.bai@nxp.com>
(cherry picked from commit 4d40b3a6149e53f60f3cc6a14da1f2ffc55efb8e)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6sll.dtsi |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sll.dtsi b/arch/arm/boot/dts/imx6sll.dtsi
index b2036fe..c722329 100644
--- a/arch/arm/boot/dts/imx6sll.dtsi
+++ b/arch/arm/boot/dts/imx6sll.dtsi
@@ -51,14 +51,14 @@
 			next-level-cache = <&L2>;
 			operating-points = <
 				/* kHz    uV */
-				996000  1225000
+				996000  1275000
 				792000  1175000
 				396000  1075000
 				198000	975000
 			>;
 			fsl,soc-operating-points = <
 				/* ARM kHz      SOC-PU uV */
-				996000          1225000
+				996000          1275000
 				792000          1175000
 				396000          1175000
 				198000		1175000
-- 
1.7.9.5

